docs/manual/scala/guide/broker/KafkaClient.md

```diff
@@ -0,0 +1,76 @@
+# Lagom Kafka Client
+
+Lagom provides an implementation of the Message Broker API that uses Kafka. In the remainder you will learn how to add the dependency in your build, and how to configure and tune topic's publishers and subscribers.
+
+## Dependency
+
+To use this feature add the following in your project's build. 
+
+In Maven:
+
+```xml
+<dependency>
+    <groupId>com.lightbend.lagom</groupId>
+    <artifactId>lagom-javadsl-kafka-broker_2.11</artifactId>
+    <version>${lagom.version}</version>
+</dependency>
+```
```

Comment:
We don't need the maven instructions in the Scala documentation, and this snippet is wrong anyway because it points to the javadsl artifact.